core/core_modules/update.js

```diff
@@ -64,10 +73,11 @@ exports.match = function(event, commandPrefix) {
 };
 
 exports.load = function() {
-	var isEnabled = this.config.getConfig('update').autoUpdateEnabled,
-        branchName;
+	var isEnabled = null,
+		branchName;
 
-    shutdown = this.shutdown;
+	isEnabled = this.config.getConfig('update').autoUpdateEnabled;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554993048/issues/source?bid=3282987&fileBranchId=3284644#l79)